Dockerfile

```diff
@@ -1,12 +1,7 @@
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
 
-RUN apt-get update && \
-    apt-get install -y curl gnupg && \
-    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
-    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && \
-    apt-get update -y && \
-    apt-get install google-cloud-sdk-gke-gcloud-auth-plugin -y && \
-    apt-get -y install kubectl
+RUN apk --update add curl && \
+    apk add kubectl
 
 RUN curl -LO \
```

Comment:
General question - not related directly to this PR, but I couldn't spot helm being used via this image in eQ. Is this binary still needed?

Reply:
I don't think it is need, I removed it from the base image and it seems to read secrets without any issues, maybe be worth anyone reviewing to test it too?

Reply:
This image is used for more than just the secrets, it is used as the base image for deploying benchmark which uses helm, hence we can't remove helm.

Reply:
Thanks for confirming 👍